src/components/header.js

```diff
@@ -98,26 +97,20 @@ const useStyles = makeStyles(theme => ({
   }
 }));
 
-export function Header(props) {
-  // eslint-disable-next-line
-  // const [auth, setAuth] = useState(false);
-  const [anchorEl, setAnchorEl] = useState(null);
+export const Header = props => {
   const [openDrawer, setOpenDrawer] = useState(false);
-  const open = Boolean(anchorEl);
+  const open = Boolean(false);
```

Comment:
just `const open = false`?

but also, if its always false, and is a const... do we need it at all?